### PR TITLE
[10.2] [PR 1995] Add missing cron job

### PR DESCRIPTION
**Pull request by settermjd**
Backport of PR #1995

### CODE REVIEW

**Comment by settermjd** [comment]
@pako81, can you review this PR pls?

File: modules/admin_manual/pages/installation/ubuntu_18_04.adoc [hunk 1]
```diff
@@ -160,6 +160,8 @@ occ config:system:set trusted_domains 1 --value="$myip"
 ----
 echo "*/15  *  *  *  * /usr/bin/php -f {install-directory}/cron.php" \
   > /var/spool/cron/crontabs/{webserver-user}
+echo "13 0  *  *  * /usr/bin/php -f {install-directory}/occ dav:cleanup-chunks" \
+  > /var/spool/cron/crontabs/{webserver-user}
```
Comment (pako81) [inline modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: Should we document that, by default, the command cleans up chunks older than 2 days and that the number of days is actually configurable as input option?

Comment (settermjd) [reply modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: Sounds like a good idea. I'll get that done.

Comment (settermjd) [reply modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: Honestly, I'm not sure what that wasn't added, originally, as part of this PR. Adding it now. Thanks again, @pako81.

Comment (settermjd) [reply modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: @pako81, I just grokked the docs and found that the relevant information's already documented in `occ_command.adoc` and `background_jobs_configuration.adoc.`

**Pull request merged**